elfi/examples/gauss_nd.py

```diff
@@ -0,0 +1,130 @@
+"""The general, n-dimensional Gaussian noise model."""
+
```

Comment:
Maybe put this inside `gauss.py`, since pretty much everything but the simulator is the same? Similar to the Ricker model, you can have an argument for `get_model` select which simulator to use.